lib/client/jsx/actions/magma_actions.js

```diff
@@ -106,17 +114,17 @@ export const requestDocuments = (args)=>{
         return;
       }
 
-
-      e.response.json().then((response)=>{
-        let errStr = response.errors.map((error)=> `* ${error}`);
+      e.response.json().then((response) => {
+        let errStr = response.error
+          ? response.error
+          : response.errors.map((error) => `* ${error}`);
         errStr = [`### Our request was refused.\n\n${errStr}`];
         dispatch(showMessages(errStr));
-      });
 
-      if(error != undefined){
```

Comment:
This seems like it never should have worked, since the `error` callback wouldn't have a `response` method or property associated with it...so fixed this up.